apertium-tur-uzb.tur-uzb.dix

```diff
@@ -361,7 +365,7 @@
     <e><p><l>dans<s n="n"/></l><r>raqs<s n="n"/></r></p></e>
     <e><p><l>raks<s n="n"/></l><r>raqs<s n="n"/></r></p></e>
     <e><p><l>taraf<s n="n"/></l><r>tomon<s n="n"/></r></p></e>
-    <e><p><l>sever<s n="n"/></l><r>sevor<s n="n"/></r></p></e>
+    <e><p><l>sever<s n="n"/></l><r>sevar<s n="n"/></r></p></e>
```

Comment:
Is this really a noun?  Could you give an example?

Reply:
I was trying to correct the word translation, didn't pay attention to it's tag.
It's not an actual word right?, it's a suffix, noun+sevar = adj, so this is "adj"?

Reply:
Which word are you trying to translate?  I don't understand.

Reply:
The word "sever", I mean, it does not come alone in a sentence, right? It comes in forms of "elksever", "milletsever","...sever",...

Reply:
sever it is a verb, it is aorist,  you need to add a stem into bidix which is "sev" v iv.
vatansever, milletsever, or iyiliksever could be add stem to bidix  adjective and noun, you should not separate them as "sever" and "vatan"

Reply:
Ok, since sever is not a lemma and verb sev already exists in both lexc and bidixes properly, I deleted sever from tur.lexc, sevar from uzb.lexc as well as sever-sevar from bidix.